src/ui/molecules/header/header.scss

```diff
@@ -17,14 +17,20 @@
 
   @media screen and (max-width: 360px) {
     grid-template-areas:
-      'logo'
+      'firstElement'
       'themeswitch';
     justify-items: center;
     row-gap: 10px;
   }
 
-  .okp4-header-logo-container {
-    grid-area: logo;
+  .okp4-header-first-element-container {
+    grid-area: firstElement;
+
+    :first-child {
+      max-width: 270px;
+      max-height: 72px;
+      display: flex;
```

Comment:
What's the purpose of a `flex` display here?
We cannot deduce from the positioning of the elements passed as component property, it must remain the responsibility of the parent.

Reply:
The purpose is that the container is too small to realistically have another layout. But as you say we shouldn't deduce the layout of the property so I will remove this